en/config/actuators.md

```diff
@@ -1,65 +1,170 @@
+
+:::note
+Not all aspects of a geometry are necessarily configurable through this interface.
+Factors that cannot be configured are hard coded for the current airframe. 
+:::
+
+### Motor Geometry
+
+The motor geometry section lets you set the number motors, and the relative position, function, and other properties of each motor.
+The motor properties are similar for all frames, but there are some exceptions.
+
+Multicopter frames (only) provide a diagram showing the X, Y positions, which is useful for checking that the motors configuration is correct, and that the motors rotate in the expected direction.
+To get a feeling for the motor layout for VTOL and other vehicles see the [Airframe Reference](../airframes/airframe_reference.md)
+
+The motor geometry sections for a number of different frames are shown below.
 
-The screenshot below shows the geometry screen for a multicopter frame, which allows you to select the number of motors, their relative positions on the frame, and their expected spin directions (select "**Direction CCW**" for counter-clockwise motors).
-This particular frame also includes an image showing the motor positions, which dynamically updates as the motors settings are changed.
+
+#### Motor Geometry: Multicopter
+
+The screenshot below shows the geometry screen for a multicopter frame with (right) and without (left) advanced settings.
+Multicopters don't have any control surfaces, so none are shown.
 
 ![Geometry MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_geometry.png)
 
-A fixed wing airframe would instead display the parameters that define control surfaces, while a VTOL airframe could display both motors and control surfaces.
+The **Motors** setting lets you choose the number of motors.
 
+For each multicopter motor you can then set:
+- `Position X`: [X-position](#motor-position-coordinate-system), in metres.
+- `Position Y`: [Y-position](#motor-position-coordinate-system), in metres.
+- `Position Z`: [Z-position](#motor-position-coordinate-system), in metres.
+- `Direction CCW`: Checkbox to indicate motor spins counter-clockwise (uncheck for clockwise).
+- `Bidirectional`: Checkbox to indicate motor is [bidirectional](#bidirectional-motors) 
+- `Slew Rate`: ?
 
-### Position Coordinate system
+The X, Y, Z positions are in [FRD relative to the centre of gravity](#motor-position-coordinate-system).
 
-The coordinate system is FRD (in body frame), where the X axis points forward, the Y axis to the right and the Z axis down.
 
-:::note
-**Positions are relative to the Center of Gravity of the vehicle, and NOT the Autopilot location!**
-:::
+#### Motor Geometry: VTOL Tailsitter
+
+The default tailsitter motor geometry is shown below.
+Motors have the same configuration fields as for the [multicopter geometry](#motor-geometry-multicopter).
+
+![Geometry motor: tailsitter vtol](../../assets/config/actuators/qgc_geometry_tailsitter_motors.png)
```

Comment:
I think you are referring to the Quad Tailsitter (https://docs.px4.io/master/en/airframes/airframe_reference.html#vtol-quad-tailsitter). But there also is the Duo Tailsitter (https://docs.px4.io/master/en/airframes/airframe_reference.html#vtol-duo-tailsitter), which is what I first thought of when I read the name "VTOL Tailsitter".

It would be better to clarify that this is a 'Quad' Tailsitter!